filebeat/docs/howto/migrate-to-filestream.asciidoc

```diff
@@ -0,0 +1,243 @@
+[[migrate-to-filestream]]
+== Migrate log input configurations to filestream
```

Comment:
```suggestion
== Migrate `log` input configurations to `filestream`
```

Reply:
In most of the docs, we use backticks to format input names (this helps map the description to the keywords used in the config) so I'm doing that here for consistency.